src/Twig/Handler/ImageHandler.php

```diff
@@ -29,16 +29,16 @@ public function __construct(Silex\Application $app)
     /**
      * Helper function to make a path to an image.
      *
-     * @param string         $filename Target filename
-     * @param string|integer $width    Target width
-     * @param string|integer $height   Target height
-     * @param string         $crop     String identifier for cropped images
+     * @param string  $filename Target filename
+     * @param integer $width    Target width
+     * @param integer $height   Target height
+     * @param string  $crop     String identifier for cropped images
      *
      * @return string Image path
      */
-    public function image($filename, $width = '', $height = '', $crop = '')
+    public function image($filename = '', $width = 0, $height = 0, $crop = '')
```

Comment:
Ah, right. My thinking about the defaults was mostly to syncronize all the functions, previously image and thumbnail defaulted to empty string, popup defaulted to 100 and showImage defaulted to 0. Since the value is expected to be an integer this seemed logical.

popup and showImage defaulted to empty string for filename, so I went with that.

I'll remove the defaults from them all :)